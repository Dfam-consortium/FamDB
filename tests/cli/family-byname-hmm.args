family
Test family TEST0003
-f
hmm
