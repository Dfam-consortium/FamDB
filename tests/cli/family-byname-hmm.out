<model3>
MS    TaxId:3 TaxName:Other_Order
CC    RepeatMasker Annotations:
CC         Type: 
CC         SubType: 
CC         Species: Other_Order
CC         SearchStages: 30,40
CC         BufferStages: 
